akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -2193,15 +2354,19 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
         replyTo ! g
       }
     }
-    val myMissingKeys = otherKeys.diff(myKeys)
+    val myMissingKeys =
+      if (expiryEnabled)
+        otherKeys.diff(myKeys)
+      else
+        otherKeys.diff(myKeys)
```

Comment:
`expiryEnabled` is no longer different

```suggestion
    val myMissingKeys = otherKeys.diff(myKeys)
```